metricshub-doc/src/site/markdown/release-notes.md

```diff
@@ -5,6 +5,81 @@ description: Learn more about the new features, changes and improvements, and bu
+| [**\#109**](https://github.com/MetricsHub/enterprise-connectors/issues/109) | **Brocade SAN Switch**: The `bandwidth` attribute incorrectly displays the WWN instead of the actual network bandwidth                                                                                                                                                 |
+| [**\#104**](https://github.com/MetricsHub/enterprise-connectors/issues/104) | **HPE OneView (Frames and Blades)**: The CPU collect fails due to a missing REST API URL                                                                                                                                                                               |
+| [**\#106**](https://github.com/MetricsHub/enterprise-connectors/issues/106) | **Palo Alto Firewall (SNMP)**: The `firewall.sessions` metric reports the protocol attribute as `ucp` instead of `udp`                                                                                                                                                 |
+
+### MetricsHub Community Edition v1.0.08
+
+#### What's New
+
+| ID                                                                         | Description                                                                                                                                                                                                                               |
+| -------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**\#765**](https://github.com/MetricsHub/metricshub-community/issues/765) | Added [MCP tool](./integrations/ai-agent-mcp.md) to get host details                                                                                                                                                                      |
+| [**\#772**](https://github.com/MetricsHub/metricshub-community/issues/772) | Custom monitoring jobs in `metricshub.yaml` can now reference resource attributes using the syntax `${esc.d}{resource.attribute::ATTRIBUTE-KEY}`, where ATTRIBUTE-KEY is configured at the resource level                                 |
+| [**\#773**](https://github.com/MetricsHub/metricshub-community/issues/773) | Custom monitoring jobs in `metricshub.yaml` can now reference protocol properties using the syntax `${esc.d}{protocol::PROTOCOL.PROPERTY}`, where PROTOCOL is the name of the protocol and PROPERTY is the specific property to reference |
+| [**\#796**](https://github.com/MetricsHub/metricshub-community/issues/796) | Added support for JWT authentication in the REST API                                                                                                                                                                                      |
```

Comment:
@NassimBtk : why did we add support for JWT authentication?

Reply:
We added JWT authentication to strengthen security and session handling. It allows stateless authentication, and enables the MCP and future Web UI to support login/logout flows without relying solely on API keys.